src/modules/Builder/features/AboutApplet/AboutApplet.tsx

```diff
@@ -21,7 +21,7 @@ import { colorThemeOptions } from './AboutApplet.const';
 const commonUploaderProps = {
   width: 20,
   height: 20,
-  maxFileSize: MAX_FILE_SIZE_1GB,
+  maxFileSize: MAX_FILE_SIZE_5MB,
```

Comment:
Can you check if there are  the same restrictions for images in Activity About Page?